components/Form.tsx

```diff
@@ -1,8 +1,47 @@
+"use client"
+
 import Image from "next/image";
 import DogImage from "../public/Form images/dog_form.png";
 import CatImage from "../public/Form images/cat_form.png";
+import { FormEvent } from "react";
+import { useGoogleReCaptcha } from "react-google-recaptcha-v3";
 
 const Form = () => {
+  const { executeRecaptcha } = useGoogleReCaptcha();
+
+  const handleSubmit = async (e: FormEvent) => {
+    e.preventDefault();
+
+    if (!executeRecaptcha) {
+      console.log("Not available to execute reCAPTCHA");
+      return;
+    }
+
+    const gRecaptchaToken = await executeRecaptcha("formSubmit");
+
+    try {
+      const response = await fetch("/api/route", {
+        method: "POST",
+        headers: {
+          Accept: "application/json, text/plain, */*",
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ gRecaptchaToken }),
+      });
+
+      const responseData = await response.json();
+
+      if (responseData?.success === true) {
```

Comment:
You can simplify this check to `responseData?.success`. The `=== true` check is redundant.